res/edid/expected/ADA-HDMI-ee5016cd008fc20c86079510a8e3f132.expected.json

```diff
@@ -0,0 +1,365 @@
+{
+    "edid": {
+        "preferred_group": "DMT",
+        "preferred_mode": 27,
```

Comment:
Hmm, because I took this value from the RPi dump:

```
...
HDMI:EDID found preferred DMT detail timing format: 1280x800p @ 60 Hz (27)
...
```

but you're right, that would normally be DMT 28.

What should the contents of this whole `.expected.json` be then?

Reply:
I think the .expected.json should say that the ADA-HDMI one for the SK should be DMT, 28 as set in the config. Because of the setting in the config text, we don't expect to run that for the said SK, but you never know

Reply:
To clarify, the `optimal_mode` should be DMT, 28 - the rest can be whatever the EDID reports, in this case DMT, 27